satellite/console/usercredits.go

```diff
@@ -19,19 +21,19 @@ type UserCredits interface {
 
 // UserCredit holds information about an user's credit
 type UserCredit struct {
-	ID                   int
-	UserID               uuid.UUID
-	OfferID              int
-	ReferredBy           uuid.UUID
-	CreditsEarnedInCents int
-	CreditsUsedInCents   int
-	ExpiresAt            time.Time
-	CreatedAt            time.Time
+	ID            int
+	UserID        uuid.UUID
+	OfferID       int
+	ReferredBy    uuid.UUID
+	CreditsEarned rewards.USD
```

Comment:
This struct is not for display purpose. Should we keep it as `rewards.Cent` ?

Reply:
So `USD` is not for just display, but for representing money such you cannot misconvert to wrong units. However there might be a better place to store `USD` type than package `rewards`.

Reply:
A-ha, I think after I read the whole PR I understand the purpose now. Maybe just to me, but `USD` equals `dollars` in terms of unit for money. So it's a little confusing for me when I read this line